test/evergreen.yml

```diff
@@ -5584,7 +5584,7 @@ buildvariants:
       WT_BUILDDIR=$WT_TOPDIR/cmake_build
       ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0:unmap_shadow_on_exit=1"
       LSAN_OPTIONS="print_suppressions=0:suppressions=$WT_TOPDIR/test/evergreen/asan_leaks.supp"
-      ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer
+      ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v4/bin/llvm-symbolizer
```

Comment:
This will change our version of `llvm-symbolizer` from `10.0.0` to `12.0.1`, and the new name `MongoDB LLVM version 12.0.1` implies we may have a custom binary. If all our UBSAN/ASAN tests are running correctly I think we're ok, but do we know what changes we'll see from this upgrade?    

```
ubuntu@ip-10-122-10-193:~$ /usr/lib/llvm-10/bin/llvm-symbolizer --version
LLVM (http://llvm.org/):
  LLVM version 10.0.0
  Optimized build.
  Default target: x86_64-pc-linux-gnu
  Host CPU: skylake-avx512

ubuntu@ip-10-122-10-193:~$ /opt/mongodbtoolchain/v4/bin/llvm-symbolizer --version
llvm-symbolizer
MongoDB LLVM version 12.0.1
  Optimized build.
  Default target: x86_64-mongodb-linux
  Host CPU: skylake-avx512
```